src/code/z_kaleido_setup.c

```diff
@@ -1,9 +1,144 @@
 #include "global.h"
+#include "overlays/gamestates/ovl_file_choose/z_file_choose.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_kaleido_setup/func_800F4A10.s")
+s16 sKaleidoSetupKscpPos[] = { PAUSE_1, PAUSE_2, PAUSE_3, PAUSE_0 };
```

Comment:
I know we're matching OoT here, but I think it's pretty unnecessary to abbreviate so much on a static variable that's used once, especially when it pointlessly obscures... whatever this does.

Reply:
Yeah, I'm not sure how I feel about this variable. I'm considering unnaming it

Reply:
Looks like OoT might rename it soon anyway, wouldn't mind not naming it for now.

Reply:
Yeah, I unnamed it already